man/review_hotspots.Rd

```diff
@@ -6,7 +6,8 @@
 \usage{
 review_hotspots(
   annotated_maf,
-  genes_of_interest = c("FOXO1", "MYD88", "CREBBP", "NOTCH1", "NOTCH2", "CD79B", "EZH2"),
+  genes_of_interest = c("FOXO1", "MYD88", "CREBBP", "NOTCH1", "NOTCH2", "CD79B",
+    "EZH2"),
```

Comment:
Was this change manually? my `devtools::document()` revert it.

Reply:
No, this was created when I run the devtools::document. My guess it is something with settings, and probably that line is more than 80 characters so got transferred to another row